### PR TITLE
Add proxy support

### LINKED ISSUE #68: Proxy support
Proxy support was forgotten when reimplementing Mopidy-Spotify for the 2.0 release.

Ref. mopidy/pyspotify#176.

### PR DESCRIPTION
**Pull request by jodal**
Fixes #68

### CODE REVIEW

File: mopidy_spotify/backend.py [hunk 1]
```diff
@@ -89,13 +89,30 @@ def _get_session(self, config):
...
+
         if config['spotify']['allow_cache']:
             spotify_config.cache_location = ext.get_cache_dir(config)
         else:
             spotify_config.cache_location = None
+
         spotify_config.settings_location = ext.get_data_dir(config)
+
+        proxy_uri = None
+        if config['proxy'].get('hostname'):
```
Comment (adamcik) [inline mopidy_spotify/backend.py]: Why not use `httpclient.format_proxy`?

Comment (adamcik) [inline mopidy_spotify/backend.py]: That is with `auth=False`

Comment (jodal) [inline mopidy_spotify/backend.py]: That's a good idea... the current code was just stolen from Mopidy-Spotify 1.x.

**Comment by jodal** [comment]
Updated PR.

**Pull request merged**